balancer-labs/frontend-v2
# hotfix: add chip to new gauges

# Description

Adds `addedTimestamp` - time the gauge was added to the controller - to the voting JSON and displays BalChipNew if it's less than 2 weeks ago.

## Type of change


## How should this be tested?

Visit veBAL page and check if the recently created gauges (e.g. bbaUSD, maticX, and stMATIC) are labeled with the new tag.

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:
